### PR TITLE
Add version command

### PR DESCRIPTION
**Pull request by KeisukeYamashita**
## What

I added `kayentactl version` command.

## Why

Solves https://github.com/armory-io/kayentactl/issues/35

### CODE REVIEW

File: cmd/version/version.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	"github.com/spf13/cobra"
+)
+
+// versionCmd represents the version command
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Prints kayentactl version",
+	Long:  ``,
+	Run: func(cmd *cobra.Command, args []string) {
+		cmd.Printf("kayentactl has version %s built from %s on %s\n", version.Version, version.Commit, version.Date)
```
Comment (ethanfrogers) [inline cmd/version/version.go:29]:
```suggestion
		cmd.Printf("kayentactl version %s, build %s, date %s", version.Version, version.Commit, version.Date)
```

Comment (ethanfrogers) [reply cmd/version/version.go:29]: Slight wording change.

**Review by ethanfrogers — CHANGES_REQUESTED** [review]
@KeisukeYamashita  - thanks for adding this! I left a small suggestion to change the wording of the version message.

File: cmd/version/version.go (hunk 1 above)
Comment (KeisukeYamashita) [reply cmd/version/version.go]: thanks for your kind review！

**Review by ethanfrogers — APPROVED** [review]

**Pull request merged**